src/programs/state/massachusetts/snap.js

```diff
@@ -421,13 +425,14 @@ export { getSNAPBenefits };
 
 
 
-
-
-
-
+// REVIEW: Can this dead code be removed?
```

Comment:
No, the code is actually being changed to use 'getGovData.js', etc. We may end up dong away with 'helperFunctions.js', though. 'getGovData.js' will also be used to replace other functionality in here. We've found it to be a common data pattern.

Reply:
Okay, sounds good. Is there an issue yet for this change? If not, can we create one so that we've got this change marked on the horizon?

Reply:
It is already a pull request as we speak, I believe!

Reply:
But we can also make it an issue.